src/models/subscription.ts

```diff
@@ -153,6 +153,22 @@ export class Subscription extends Model {
 		});
 	}
 
+	static async findForRepoNameAndOwner(repoName: string, repoOwner: string, jiraHost: string): Promise<Subscription | null> {
+		const results = await this.sequelize?.query(
+			"SELECT * " +
+			"FROM \"Subscriptions\" s " +
+			"LEFT JOIN \"RepoSyncStates\" rss on s.\"id\" = rss.\"subscriptionId\" " +
+			"WHERE s.\"jiraHost\" = :jiraHost " +
+			"AND rss.\"repoName\" = :repoName " +
+			"AND rss.\"repoOwner\" = :repoOwner",
+			{
+				replacements: { jiraHost, repoName, repoOwner },
+				type: QueryTypes.SELECT
+			}
+		);
+		return results[0] as Subscription;
```

Comment:
if `this.sequelize` is undefined, this will explode in our faces

Reply:
I was using just `sequelize` initially, but then saw `this.sequelize` being used in `syncStatusCounts` method, so followed the same convention.
Will it be undefined though? Not sure of the proper way here.

Reply:
I think you could just use `this.sequelize!` as we pass it into ctor

Reply:
Updated to `this.sequelize!`